reference/docs-conceptual/install/Installing-PowerShell-Core-on-macOS.md

```diff
@@ -26,7 +27,7 @@ There are several ways to install PowerShell on macOS. Choose one of the followi
 After installing PowerShell, you should install [OpenSSL](#installing-dependencies). OpenSSL is
 needed for PowerShell remoting and CIM operations.
 
-## Installation of latest stable release via Homebrew on macOS 10.12 or higher
+## Installation of latest stable release via Homebrew on macOS 10.13 or higher
```

Comment:
This comes back to the "10.13 is only for 7.1, not 7".  We should probably hold this PR for Sean to review.

Reply:
The reason I left it is this section is about the latest stable release. Both changes are baout the latest version and moving forward, this is correct.